infra/cifuzz/cifuzz.py

```diff
@@ -92,6 +96,49 @@ def checkout_specified_commit(build_repo_manager, pr_ref, commit_sha):
         'Using current repo state', pr_ref or commit_sha)
 
 
+def get_project_src_path():
+  """Returns the manually checked out path of the project's source if specified
+  or None."""
+  # TODO(metzman): Get rid of MANUAL_SRC_PATH when Skia switches to
+  # project_src_path.
+  path = os.getenv('PROJECT_SRC_PATH', os.getenv('MANUAL_SRC_PATH'))
+  if path is None:
+    return path
+  return get_abs_src_path(path)
+
+
+def get_abs_src_path(src):
+  """Returns the absolute path of the source code repo."""
+  if os.path.isabs(src):
+    return src
+  workspace = os.environ['GITHUB_WORKSPACE']
+  abs_src_path = os.path.join(workspace, src)
+  return abs_src_path
+
+
+@retry.wrap(_IMAGE_BUILD_TRIES, _IMAGE_BUILD_BACKOFF,
+            'cifuzz.cifuzz.build_external_project_docker_image')
+def build_external_project_docker_image(project_name, project_src,
+                                        build_integration_path):
+  """Builds the project builder image for an external (non-OSS-Fuzz) project.
+  Returns True on success."""
+  dockerfile_path = os.path.join(build_integration_path, 'Dockerfile')
+  tag = 'gcr.io/oss-fuzz/{project_name}'.format(project_name=project_name)
```

Comment:
Should we tag this with a different prefix?

Reply:
+1

Reply:
Do people object to this reasoning? If so I can leave a TODO.